hooks/boot/13-clear_pulpcore_content.rb

```diff
@@ -0,0 +1,8 @@
+if pulpcore_enabled?
```

Comment:
Being in boot phase, this will need to use the new `module_present?` helper (https://github.com/theforeman/kafo/pull/262) and likely introduce a `pulpcore_present?` to also help draw the distinction.